sweepai/handlers/on_comment.py

```diff
@@ -3,6 +3,18 @@
 import openai
 from loguru import logger
 
+def construct_metadata(repo_full_name, repo_name, organization, repo_description, installation_id, username, function, mode):
```

Comment:
Sweep: Move construct metadata to the bottom of the file next to posthog_capture